src/Mages.Core/Runtime/GlobalScope.cs

```diff
@@ -3,17 +3,46 @@
     using Mages.Core.Runtime.Functions;
     using System;
     using System.Collections.Generic;
-
-    sealed class GlobalScope : BaseScope
+    using System.Reflection;
+    sealed class GlobalScope : Scope
     {
-        public GlobalScope(IDictionary<String, Object> scope)
-            : base(scope ?? new Dictionary<String, Object>(), new Dictionary<String, Object>(Global.Mapping))
+        void LoadType(Type type)
+        {
+            foreach (var f in type.GetFields(BindingFlags.Public | BindingFlags.Static))
+                _scope.Add(f.Name.ToLower(), f.GetValue(null));
+        }
+
+        public GlobalScope(IDictionary<String, Object> scope): base(null)
+        {
+            if (scope == null)
```

Comment:
Nope, too expensive. Also I still want to be able to fine-tune this. Therefore the Global.cs was there intentionally. It is *way* cheaper to lead and much easier to control.